R/lasso-development.R

```diff
@@ -216,7 +214,7 @@ LassoDevelopment <- R6Class("LassoDevelopment",
       if (self$params$type == 'classification') {
         private$fitLogit <- glm(
           as.formula(paste(self$params$predictedCol, '.', sep = " ~ ")),
-          data = private$dfTrain,
+          data = private$dfTrainRaw,
```

Comment:
Yeah, this is basically it. Creating dummies for training data is really slow and not needed. We save the factor levels in develop and assign them before dummification in deploy.